iceoryx_utils/include/iceoryx_utils/log/logger.inl

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef IOX_UTILS_LOG_LOGGER_INL
+#define IOX_UTILS_LOG_LOGGER_INL
+namespace iox
+{
+namespace log
+{
+inline bool Logger::IsEnabled(const LogLevel logLevel) const noexcept
```

Comment:
the `logger.inl` is not necessary. Please move this function to `logger.cpp`

Reply:
moved